misc-utils/uuidd.c

```diff
@@ -527,8 +527,6 @@ static void server_loop(const char *socket_path, const char *pidfile_path,
 		case UUIDD_OP_BULK_RANDOM_UUID:
 			if (num < 0)
 				num = 1;
-			if (num > 1000)
-				num = 1000;
```

Comment:
Good catch. Maybe the buffer size should be defined as number of uuids 
```
#define UUIDD_PROT_BUFSZ  (63 * sizeof(uuid_t))
```
or so.